tests/tests_change_settings.yml

```diff
@@ -0,0 +1,109 @@
+- hosts: all
+  tasks:
+    - name: Ensure required packages are installed
+      package:
+        name: tuned
+        state: present
+
+    - name: Ensure kernel settings profile directory exists
+      file:
+        path: /etc/tuned/kernel_settings
+        state: directory
+        mode: 0755
+
+    - name: Generate a configuration for kernel settings
+      copy:
+        src: tuned/etc/tuned/basic_settings/tuned.conf
+        dest: /etc/tuned/kernel_settings/tuned.conf
+        mode: 0644
+
+    - name: Ensure required services are enabled and started
+      service:
+        name: tuned
+        state: restarted
+        enabled: yes
+
```

Comment:
Sorry - forgot that there is a missing step here - to apply the kernel_settings profile:
```yaml
    - name: apply kernel_settings tuned profile
      command: tuned-adm profile kernel_settings
```

You might also want to add a check here to make sure your settings have been applied correctly e.g. `tuned-adm verify -i` or checking individual `sysctl`, `sysfs`, etc. settings.  Note that if your profile sets `[bootloader]` `cmdline` settings, `tuned-adm verify -i` will report failure unless you first reboot the system.